ecchronos-binary/pom.xml

```diff
@@ -105,6 +105,12 @@
                                 <includedLicense>GPL2 w/ CPE</includedLicense>
                                 <includedLicense>Public Domain, per Creative Commons CC0</includedLicense>
                                 <includedLicense>BSD-2-Clause</includedLicense>
+                                <includedLicense>The MIT License (MIT)</includedLicense>
```

Comment:
Would be easier to sort this so that all the licenses which are the same thing are grouped.